source/confirm_conditions.md.erb

```diff
@@ -0,0 +1,35 @@
+---
+title: Confirm conditions
+weight: 50
+---
+
+# Confirm conditions
+Confirm that candidate has met conditions.
+
+```
+POST /applications/:uuid/confirm_conditions
```

Comment:
🤔 I'm thinking I would prefer:

```suggestion
POST /applications/:uuid/confirm_conditions_met
```

When I think `confirm conditions`, I think "is this to confirm that I (as a provider) am happy make an offer with these conditions" or "is this to confirm that the candidate has met the conditions of our offer".

Would be interesting to hear the thoughts of others.

Reply:
I'm happy with `confirm conditions met`. The other action you described would be `set conditions` (from the cukes)